apprentice-react-o-shop/src/components/Auth/RegisterForm.js

```diff
@@ -36,10 +39,10 @@ function RegisterForm() {
 
     const inputFieldList = [];
 
-    for (const key in formFields){
-        if (formFields[key].register){
+    formFields.forEach(inputObject => {
+        if (inputObject.register){
             var referenceVariable;
-            switch (formFields[key].name) {
+            switch (inputObject.name) {
```

Comment:
Similar here, just use `const referenceVariable = inputObject.name`. That way we can get rid of this huge switch statement. The app is simple right now so lets keep it simple.